Documentation/building/windows-instructions.md

```diff
@@ -45,7 +45,8 @@ For Visual Studio 2017:
     * Visual Studio C++ core features
   * Under "SDKs, libraries, and frameworks":
     * Windows 10 SDK or Windows 8.1 SDK
-* To build for Arm32, Make sure that you have the Windows 10 SDK installed (or selected to be installed as part of VS installation). To explicitly install Windows SDK, download it from here: [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads)
+* To build for Arm32, Make sure that you have the Windows 10 SDK installed (or selected to be installed as part of VS installation). To explicitly install Windows SDK, download it from here: [Windows SDK for Windows 10](https://developer.microsoft.com/en-us/windows/downloads).
```

Comment:
There are two separate boxes for arm and arm64.

Reply:
Probably best not to mention ARM64 for windows unless/until the build is supported.